### PR TITLE
Create BIBLIOGRAPHY.md

### PR DESCRIPTION
**Pull request by ftyers**
Here are a couple of papers I found that use DeepSpeech. We can add more as and when we find them!

### CODE REVIEW

File: BIBLIOGRAPHY.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+This file contains a list of papers in chronological order that have been published 
+using Mozilla's DeepSpeech.
```
Comment (lissyx) [inline BIBLIOGRAPHY.md]: nit: bibtex entry to cite deepspeech?

Comment (ftyers) [reply BIBLIOGRAPHY.md]: @lissyx We don't have a DeepSpeech paper yet! When the CommonVoice paper is published we can cite that too.

Comment (lissyx) [reply BIBLIOGRAPHY.md]: That might still be useful for people to refer to us

**Comment by reuben** [comment]
Some more:

https://arxiv.org/abs/2002.03520

**Comment by reuben** [comment]
Sorry I don't know how I messed up making a comment. Added four more references that I know of. Thanks for starting this PR!

File: BIBLIOGRAPHY.md (hunk 1 above)
Comment (reuben) [reply BIBLIOGRAPHY.md]: Should we include the Common Voice preprint here already and then update it once it's published?

Comment (ftyers) [reply BIBLIOGRAPHY.md]: I think that's a good idea, I've added a section for "To appear" where we can put submitted/accepted but not yet published papers.

**Comment by ftyers** [comment]
> Some more:
> 
> https://arxiv.org/abs/2002.03520
> 
> https://krex.k-state.edu/dspace/handle/2097/39830
> 
> https://ieeexplore.ieee.org/abstract/document/8638637
> 
> http://repositori.usu.ac.id/handle/123456789/15447

I've added these. One of them is in Indonesian and I can't work out what kind of thesis it is so I've just put masters. 

The ones that are "submitted" but not "published" I've put in "to appear".

**Pull request merged**